docs/whats_new.rst

```diff
@@ -23,6 +23,7 @@ Current (0.7.dev0)
 
 Enhancements
 ~~~~~~~~~~~~
+- Adding EEG-ITNet Network :class:`braindecode.models.EEGITNet` (:gh:`400` by `Ghaith Bouallegue`_)
```

Comment:
I suspect you also need to add your name at the bottom of this file

Reply:
thank you ! it is done ☺️